app/workers/go_project_verification_worker.rb

```diff
@@ -77,7 +77,7 @@ def perform(name)
   # if the proxy sends back the same cased name that we sent, then it will likely do that for all the different casings
   # in that case downcase the name and use that as the canonical name for this package
   def non_module_name(name)
-    proxy_name = PackageManager::Go.canonical_module_name(name)
+    proxy_name = PackageManager::Go.name_in_go_mod(name)
```

Comment:
This certainly reads as bizarre - "to get the non module name, get the name from go mod"

Does it make sense in some kind of way or is it actually bizarre?

Reply:
I mean, the description above seems to describe the behavior, but agree that the name of the method seems 🤔

Reply:
what is the actual intention of this method I'm wondering

Reply:
the comment above the method says it queries the proxy but it doesn't. maybe it does back in git history?

Reply:
This was introduced in https://github.com/librariesio/libraries.io/pull/2916/files#diff-ae9bb5f10cefc011cb036fd16a1dd1aaceb074d0f0f72e99921436e6720f3702R37 to deal with non Go module packages.

Reply:
maybe "actual_go_module_name"  would be better name?

Reply:
Will change when we get a better handle on all the Go package name stuff in the future, for now leaving it to cut scope on this PR.